tests/gis_tests/test_spatialrefsys.py

```diff
@@ -51,7 +49,7 @@
 
 
 @skipUnlessDBFeature("has_spatialrefsys_table")
-class SpatialRefSysTest(unittest.TestCase):
+class SpatialRefSysTest(TestCase):
```

Comment:
I suppose you didn't use `SimpleTestCase` because the test was performing queries?